### DISCUSSION TITLE
REST API for creating a personal access token

### DISCUSSION THREAD

**Discussion by toadative** [community/community#24732] [Apps, API and Webhooks]
<p>Is it possible to create a personal access token via the REST API?</p>
<p>I have not been able to find an endpoint for it in the docs despite the octokit.rb docs suggesting that this is possible with <a href="https://octokit.github.io/octokit.rb/Octokit/Client/Authorizations.html#create_authorization-instance_method" rel="noopener nofollow ugc">Octokit::Client::Authorizations</a>. Indeed I get a 404 when I try to use it, so maybe this endpoint was removed? There is also <a href="https://github.community/t/rotate-personal-access-tokens/13535">a post here on the forum</a> that uses it.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>It has been removed in line with disabling password authentication for the API, see here: <a href="https://developer.github.com/changes/2020-02-14-deprecating-oauth-auth-endpoint/" class="inline-onebox">Deprecating OAuth Authorization API | GitHub Developer Guide</a></p>
